types/leaflet/index.d.ts

```diff
@@ -1541,6 +1542,11 @@ export interface PanOptions {
 // This is not empty, it extends two interfaces into one...
 export interface ZoomPanOptions extends ZoomOptions, PanOptions {}
 
+export interface InvalidateSizeOptions extends ZoomPanOptions {
```

Comment:
Can you provide specific reference to those options?

Reply:
Sure, go to https://leafletjs.com/reference-1.6.0.html and scroll down to invalidateSize().

It says : 

> If options.pan is false, panning will not occur. If options.debounceMoveend is true, it will delay moveend event so that it doesn't happen often even if the method is called many times in a row

Reply:
:+1: